src/amd.h

```diff
@@ -39,15 +39,15 @@ enum amd_led_interfaces {
 
 extern enum amd_led_interfaces amd_interface;
 
-enum amd_platforms {
+enum amd_ipmi_platforms {
 	AMD_PLATFORM_UNSET,
 	AMD_PLATFORM_ETHANOL_X,
 	AMD_PLATFORM_DAYTONA_X,
-	AMD_PLATFORM_GRANDSTAND,
-	AMD_PLATFORM_SPEEDWAY,
 };
 
-extern enum amd_platforms amd_platform;
+extern enum amd_ipmi_platforms amd_ipmi_platform;
+
+extern char *platform_name;
```

Comment:
It is used only in errors messages. This is not really needed anywhere.
Instead using platform name please print  "This AMD Platform does not have a defined IPMI channel\n"